packages/slice-machine/src/components/SideNav/SideNav.tsx

```diff
@@ -0,0 +1,297 @@
+import {
+  type FC,
+  type PropsWithChildren,
+  type HTMLProps,
+  type ReactNode,
+  type MouseEvent,
+  type LiHTMLAttributes,
+  type RefCallback,
+  type ForwardRefExoticComponent,
+  type RefAttributes,
+  type SVGProps,
+  type DetailedHTMLProps,
+  type MouseEventHandler,
+  useRef,
+  useCallback,
+  forwardRef,
+} from "react";
+import * as styles from "./SideNav.css";
+import LogoIcon from "@src/icons/LogoIcon";
+import OpenIcon from "@src/icons/OpenIcon";
+import ReactTooltip from "react-tooltip";
+import { IconButton } from "@prismicio/editor-ui";
+import { PlayCircleIcon } from "@src/icons/PlayCircle";
+import clsx from "clsx";
+
+export const SideNavLogo: React.FC<{
```

Comment:
❓For all React components should we add a code guideline for creating ComponentProps above the declaration?

https://www.notion.so/prismic/Slice-Machine-UI-s-Coding-Guidelines-d377c3e33faa44b3a5e0ad84a942616d?pvs=4#a7f22c0a22dd4f63a0cdae8da3346702

As you can see, the example have it, but it's not really written.

cc: @bapmrl @hypervillain